### PR TITLE
Adds promote github actions to orchestrator-bundle charms

### PR DESCRIPTION
**Pull request by saltiyazan**
# Description

Adds promote github action to each of the charms in orchestrator-bundle, that will allow the promotion of charms to `beta` and later to other channels.

# Checklist:

- [x] My code follows the [style guidelines](/CONTRIBUTING.md) of this project
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [ ] I have added tests that validate the behaviour of the software
- [x] I validated that new and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by Gmerold — CHANGES_REQUESTED** [review]
Could we have just one workflow with additional parameter specifying charm (or bundle) name?
Also what about bundle promotion? I can't see the workflow for the bundle. Was that intended?

File: orchestrator-bundle/orc8r-bundle/bundle.yaml [hunk 1]
```diff
@@ -7,17 +7,14 @@ description: |
...
   nms-nginx-proxy:
     charm: magma-nms-nginx-proxy
-    channel: edge
     scale: 1
     trust: true
   orc8r-accessd:
     charm: magma-orc8r-accessd
-    channel: edge
     scale: 1
     trust: true
```
Comment (Gmerold) [inline orchestrator-bundle/orc8r-bundle/bundle.yaml]: Do we still need `trust: true` if we're not using `edge`?

**Comment by saltiyazan** [comment]
> Could we have just one workflow with additional parameter specifying charm (or bundle) name? Also what about bundle promotion? I can't see the workflow for the bundle. Was that intended?

There is a [limitation](https://github.com/canonical/charming-actions/tree/main/release-charm#limitations) for the `releae-charm` action, it doesn't support releasing bundles :)

Now we have a single workflow with parameters :)

File: orchestrator-bundle/orc8r-bundle/bundle.yaml (hunk 1 above)
Comment (saltiyazan) [reply orchestrator-bundle/orc8r-bundle/bundle.yaml]: Fixed

Comment (Gmerold) [reply orchestrator-bundle/orc8r-bundle/bundle.yaml]: That was actually a question, not a suggestion ;)
According to [this](https://juju.is/docs/olm/trust-an-application-with-a-credential), `trust` flag has nothing to do with channels; so I reckon we need all these after all :D

Comment (saltiyazan) [reply orchestrator-bundle/orc8r-bundle/bundle.yaml]: Good point, fixed again :)

**Review by Gmerold — COMMENTED** [review]
BTW, if I understood correctly, you'll be promoting charm immediately, correct? 
If not, we should probably wait with removing `channel: edge` until we have something in `beta`.

**Comment by saltiyazan** [comment]
> BTW, if I understood correctly, you'll be promoting charm immediately, correct? If not, we should probably wait with removing `channel: edge` until we have something in `beta`.

Correct, I moved that to a separate PR so we have more control on when to merge it :)

**Review by Gmerold — APPROVED** [review]

**Review by ghislainbourgeois — APPROVED** [review]

**Pull request merged**